nbs/00_models.ipynb

```diff
@@ -1885,7 +1893,7 @@
     "    btrajectory = trajectory.get_batch(batch_idx)\n",
     "    dlmos_batch.append(spmodel.dlmos(btrajectory))\n",
     "    \n",
-    "dlmos_batch = np.array(dlmos_batch)\n",
+    "# dlmos_batch = np.array(dlmos_batch)\n",
```

Comment:
Is this another case where something's commented out because data is missing...?

Reply:
I commented this out because it was throwing a numpy error before but now I ran the tests and it's gone so I'll add it back and double check it later when I update the models module